gulp-core-build-karma/src/KarmaTask.ts

```diff
@@ -57,7 +67,8 @@ export class KarmaTask extends GulpTask<IKarmaTaskConfig> {
 
       completeCallback();
     } else {
-      const server: karma.Server = karma.Server;
+      const karma = require('karma'); // tslint:disable-line
```

Comment:
not sure what the import * as Karma above is for if you don't do const karma: typeof Karma = require('karma') for instance... 

I'm also a little surprised in general that doing this results in TS not having the require at the top but only down here/..